CMakeLists.txt

```diff
@@ -580,7 +580,16 @@ if(CURL_USE_WOLFSSL)
 endif()
 
 if(CURL_USE_GNUTLS)
-  find_package(GnuTLS REQUIRED)
+  if(NOT MSVC OR VCPKG_TOOLCHAIN)
```

Comment:
Reminder: vcpkg doesn't want packages to switch behaviour on vcpkg indicators. There is no need to block choice for conan, spack, `FetchContent`, whatever.

Reply:
So should we restore the `UNIX` condition here? Otherwise it's known to cause breakage for people trying to build curl!